app/react/kubernetes/applications/components/ConfigurationsFormSection/ConfigurationKey.tsx

```diff
@@ -18,7 +18,7 @@ type Props = {
     | string
     | string[]
     | FormikErrors<ConfigurationOverrideKey>[];
-  dataCyType: 'config' | 'secret';
+  configurationType: ConfigurationType;
```

Comment:
same question as above

Reply:
Yeah, the text (configmap or secret) is now used for a texttip as well as the dataCy for the Select component. Because it's used for a couple of things, I renamed it more generally